client/src/components/ElementEditor/Element.js

```diff
@@ -13,17 +13,32 @@ class Element extends Component {
   constructor(props) {
     super(props);
 
-    this.handleClick = this.handleClick.bind(this);
     this.handleKeyUp = this.handleKeyUp.bind(this);
+    this.handleExpand = this.handleExpand.bind(this);
+
+    this.state = {
+      previewExpanded: false,
+    };
   }
 
   /**
-   * Take the user to the GridFieldDetailForm to edit the record
+   * Expand the element to show the  preview
+   * If the element is not inline-editable, take user to the GridFieldDetailForm to edit the record
    */
-  handleClick() {
-    const { link } = this.props;
+  handleExpand() {
+    const {
+      element: { InlineEditable },
```

Comment:
maybe just import the `element` and use that` - the fact that the InlineEditable prop is capitalised is misleading, because usually capitalisation in React is used for component names, this sort of implies that it could be renderable

Reply:
Cool, I'll go for your last suggestion. But for consistency I'll do the same with ID, Title, and BlockSchema.

Reply:
Yes please, thanks!